flink-connectors/flink-connector-elasticsearch/src/test/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchSinkITCase.java

```diff
@@ -67,7 +68,11 @@ public void testTransportClientFails() throws Exception{
 
 	/**
 	 * Tests that the Elasticsearch sink works properly using an embedded node to connect to Elasticsearch.
+	 *
+	 * NOTE: This is ignored, since executing both embedded client mode and transport client mode introduces
+	 * instability for node discovery in ES 1.x. See FLINK-6867.
 	 */
+	@Ignore
```

Comment:
I would not expect it to be enabled again, at least for the Travis runs.
At the rate of the new Elasticsearch versions (there's already a 6.x now), we might also consider dropping 1.x support soon.

Makes sense, will remove it completely :)